plugin/src/shared/utils/is-next-app.ts

```diff
@@ -0,0 +1,5 @@
+import { Tree } from '@nx/devkit';
+
+export const isNextApp = (tree: Tree, projectRoot: string): boolean => {
+  return tree.exists(`${projectRoot}/next.config.js`);
+};
```

Comment:
let's combine these utils:

```suggestion
export type AppFramework = 'expo' | 'next';

export const isNextApp = (tree: Tree, projectRoot: string): boolean => {
  return tree.exists(`${projectRoot}/next.config.js`);
};

export const isExpoApp = (tree: Tree, projectRoot: string): boolean => {
  return tree.exists(`${projectRoot}/metro.config.js`);
};

export const getAppFrameworkName = (tree: Tree, projectRoot: string): AppFramework => {
  return isNextApp(tree, projectRoot) ? 'next' : isExpoApp(tree, projectRoot) ? 'expo' :  undefined;
};
```